userprofile.py

```diff
@@ -32,6 +32,11 @@ async def save(self, session: AsyncIOMotorClientSession = None):
 		import storage # avoid circular import
 		await storage.save_object(self, session=session)
 
+	def add_pokemon(self, pokemon_id):
```

Comment:
needs type annotation for `pokemon_id`
`pokemon_id` should just be `pokemon: Pokemon`